### PR TITLE
correct typo and reword sentence

### PR DESCRIPTION
**Pull request by ShalokShalom**
Correct typo and reword sentence

How is open source related to the creation of binaries by hand?

### CODE REVIEW

**Comment by ShalokShalom** [comment]
> I suggest adding feature to copy video name and description by pressing and holding like in the browser.

How is that related?

**Comment by dequadim** [comment]
I don't understand how is possible to submit a problem with an APK file.
The youtube-revanced-extended-v18.45.43-all.apk is corrupted and installation fails.

**Comment by ShalokShalom** [comment]
> I don't understand how is possible to submit a problem with an APK file. The youtube-revanced-extended-v18.45.43-all.apk is corrupted and installation fails.

Can you rephrase? I dont quite see the relevance to this patch

**Comment by dequadim** [comment]
> Correct typo and reword sentence
> 
> How is open source related to the creation of binaries by hand?

I apologize if my English skills are poor.
I had encountered a problem when installing the latest APK released, in the sense that upon launching the installation it suddenly ended, telling me that the file was invalid/corrupt.
I later uninstalled the previous version of Revanced Extended and was finally able to install the package that was previously giving me the problem.
Now let's see what will happen with the next update, that is, whether the problem will recur again or whether everything will be perfect as with previous updates.

**Comment by ShalokShalom** [comment]
> > Correct typo and reword sentence
> > How is open source related to the creation of binaries by hand?
> 
> I apologize if my English skills are poor. I had encountered a problem when installing the latest APK released, in the sense that upon launching the installation it suddenly ended, telling me that the file was invalid/corrupt. I later uninstalled the previous version of Revanced Extended and was finally able to install the package that was previously giving me the problem. Now let's see what will happen with the next update, that is, whether the problem will recur again or whether everything will be perfect as with previous updates.

You are not at the right place for this. 

This is a pull request, to add to the development. 
Look into the Revanced Discord Channel, to get support.

**Comment by dequadim** [comment]
Ok my sir.

I'm not a programmer, I'm just a poor mortal user whitout any programming skill.
I've searched how to submit my problem, but I've not found anything that permits to report a problem with an APK file.
I will never disturb you anymore, this is my promise.




________________________________________
Da: ShalokShalom ***@***.***>
Inviato: venerdì 8 dicembre 2023 20:05
A: NoName-exe/revanced-extended
Cc: dequadim; Comment
Oggetto: Re: [NoName-exe/revanced-extended] correct typo and reword sentence (PR #12)

Correct typo and reword sentence
How is open source related to the creation of binaries by hand?

I apologize if my English skills are poor. I had encountered a problem when installing the latest APK released, in the sense that upon launching the installation it suddenly ended, telling me that the file was invalid/corrupt. I later uninstalled the previous version of Revanced Extended and was finally able to install the package that was previously giving me the problem. Now let's see what will happen with the next update, that is, whether the problem will recur again or whether everything will be perfect as with previous updates.

You are not at the right place for this.

This is a pull request, to add to the development.
Look into the Revanced Discord Channel, to get support.

**Comment by ShalokShalom** [comment]
Oh, there is no reason to be sorry, I only tried to be helpful as you seemed confused. :)

Did you get help in the Chat?

**Comment by ShalokShalom** [comment]
What I am doing 😅

**Comment by NoName-exe** [comment]
I mean,both mean essentially the same thing

**Comment by datlmhg** [comment]
@NoName-exe Can you release another apk file with a grey theme?
